tensorflow/lite/micro/kernels/neg_test.cc

```diff
@@ -71,6 +71,72 @@ void TestNegFloat(std::initializer_list<int> input_dims_data,
   }
 }
 
+// input_quantization_buffer: buffer used for the quantization data
+void TestNegQuantizedInt8(float* input, int8_t* quantized_input_data,
+                          float input_min, float input_max,
+                          float* expected_output,
+                          int8_t* quantized_expected_output_data,
+                          int8_t* quantized_output_data, float output_min,
+                          float output_max,
+                          std::initializer_list<int> dimension_data) {
+  TfLiteIntArray* tensor_dims = IntArrayFromInitializer(dimension_data);
+  const int element_count = ElementCount(*tensor_dims);
+  constexpr int inputs_size = 1;
+  constexpr int outputs_size = 1;
+  constexpr int tensors_size = inputs_size + outputs_size;
+
+  // quantize input
+  std::transform(input, input + element_count, quantized_input_data,
+                 std::bind(tflite::testing::F2QS, std::placeholders::_1,
```

Comment:
Is there a way to avoid using std::bind? We're trying to reduce as much template magic as possible in TFLM but I'll leave @petewarden to make the call.

Reply:
I've replaced it with a lambda instead